librz/diff/diff.c

```diff
@@ -0,0 +1,827 @@
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2021 deroad <wargio@libero.it>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+/*
+ * Ratcliff/Obershelp Pattern Recognition algorithm applied to generic data
```

Comment:
It is possible to add documentation for a specific file in Doxygen: https://www.doxygen.nl/manual/commands.html#cmdfile
Maybe that would be appropritate here.